.azurepipelines/build-release-packages-1es.yml

```diff
@@ -0,0 +1,365 @@
+
+        - task: AzureKeyVault@2
+          inputs:
+            azureSubscription: 'AC - Dev Infra & Build Pool'
+            KeyVaultName: 'mobile-center-sdk'
+            SecretsFilter: 'appcenter-sdk-blob-storage-access-secret'
+            RunAsPreJob: false
+
+        - task: PowerShell@2
+          displayName: 'Set Release Version'
+          inputs:
+            targetType: filePath
+            filePath: ./build.ps1
+            arguments: '"version.cake" --Target="SetReleaseVersion"'
+
+        - task: cake-build.cake.cake-build-task.Cake@2
+          displayName: 'Prepare Assemblies'
+          inputs:
+            target: PrepareAssemblies
+            arguments: '--StorageAuthParams="$(appcenter-sdk-blob-storage-access-secret)"'
+            Version: 2.2.0
+
+
+        - task: CopyFiles@2
+          displayName: 'Copy Assemblies'
+          inputs:
+            SourceFolder: '$(Build.SourcesDirectory)\bin'
+            TargetFolder: '$(Build.ArtifactStagingDirectory)'
+
+    - stage: BuildDotNetAssemblies
+      dependsOn: []
+      pool:
+        name: Azure Pipelines
+        image: macos-11
+        os: macOS
+        demands:
+        - msbuild
+      jobs:
+      - job: sdkBuildJob
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact: framework'
+            path: '$(Build.ArtifactStagingDirectory)'
+            artifactName: dotnet-assemblies
+        steps:
+        - checkout: self
+          fetchTags: false
+
+        - task: Bash@3
+          displayName: 'Set Release Version'
+          inputs:
+            targetType: filePath
+            filePath: ./build.sh
+            arguments: '"version.cake"  -t SetReleaseVersion'
+
+        - bash: 'sudo $AGENT_HOMEDIRECTORY/scripts/select-xamarin-sdk.sh 6_12_0'
+          displayName: 'Use Xamarin Bundle 6.12.0'
+
+        - task: JavaToolInstaller@0
+          displayName: 'Use Java 11'
+          inputs:
+            versionSpec: 11
+            jdkArchitectureOption: x64
+            jdkSourceOption: PreInstalled
+
+        - task: AzureKeyVault@2
+          inputs:
+            azureSubscription: 'AC - Dev Infra & Build Pool'
+            KeyVaultName: 'mobile-center-sdk'
+            SecretsFilter: 'appcenter-sdk-blob-storage-access-secret'
+            RunAsPreJob: false
+
+        - task: UseDotNet@2
+          displayName: 'Install .NET 7'
+          inputs:
+            packageType: 'sdk'
+            version: '7.0.308'
+
+        - bash: 'dotnet workload install android ios macos'
+          displayName: 'Install .NET workloads'
+
+        - task: cake-build.cake.cake-build-task.Cake@2
+          displayName: 'Prepare DotNet Assemblies'
+          inputs:
+            target: PrepareAssemblies
+            arguments: '--StorageAuthParams="$(appcenter-sdk-blob-storage-access-secret)"'
+            Version: 2.2.0
+
+        - task: UseDotNet@2
+          displayName: 'Install .NET 3.1'
```

Comment:
Why do we need .net 3.1?

Reply:
Removed. It was added as part of testing on newer macos image, for installing mono and msbuild. Preinstalled on the current image.